MIP41/MIP41c4-Subproposal-Template.md

```diff
@@ -36,3 +36,12 @@ Date Ratified: <yyyy-mm-dd>
 - The Facilitator Commitment is a detailed description and plan of what the Facilitator plans to do in order to achieve success for the Core Unit. There is a lot of flexibility in what a Facilitator can write in their commitment, including organizational structure and theory, plans for the budget, hiring and infrastructure plans, etc.
 - The Facilitator commitment should also contain targets and goals that can be observed by Governance, if appropriate. This helps Governance to see whether the Facilitator is living up to their expectations.
 - A crucial part of the Facilitator Commitment are clear commitments to transparency. The Facilitator should commit to an effective plan that will allow Governance and other stakeholders, such as contributors contemplating doing work, to gain direct insight into what is being worked on and who are taking care of what responsibilities.
+
+### Succession Plan
+
+In the event this facilitator is no longer active (for any reason), state which individual person or role within the Core Unit will serve as Interim Facilitator. If this is unnecessary (e.g. the Core Unit has multiple facilitators), state the reasoning specifically.
```

Comment:
How long can a Core Unit go without a (further?) ratified Facilitator?

Reply:
good Q, I'd imagine that's specific to the budget – right?